### PR TITLE
Add travis config, postproc, disable shape anitaliasing and fix name

### PR DESCRIPTION
**Pull request by borodust**
Your CI config was correct, but you've used keyword name for as a name for defgame which confused the script - it's better to use plain symbols there. Added postproc and native downscaled canvas (200x150).

### CODE REVIEW

**Comment by borodust** [comment]
If you decide to merge this in, don't forget to fix secrets in both CI configs.

File: src/main.lisp [hunk 1]
```diff
@@ -2,14 +2,20 @@
...
 ; Set window properties
-(gamekit:defgame :moppu ()()
+(gamekit:defgame moppu (gamekit.postproc:postproc)()
   (:viewport-width 800)           ; window's width
   (:viewport-height 600)          ; window's height
   (:viewport-title "moppu")    ; window's title
   (:draw-rate 120)
-  (:act-rate 120))
+  (:act-rate 120)
+  (:default-initargs :postproc-indirect-width 200
```
Comment (borodust) [inline src/main.lisp:11]: Change this values to have native downscaled canvas of the size you need

File: src/main.lisp [hunk 2]
```diff
@@ -2,14 +2,20 @@
...
+                     :postproc-indirect-height 150))
+
+(defun run ()
```
Comment (borodust) [inline src/main.lisp:14]: To run your game, in REPL invoke:
```
(ql:quickload :moppu) (moppu::run)
```

File: src/main.lisp [hunk 3]
```diff
@@ -125,9 +131,9 @@
 (setf (coll *block-d*) (gamekit:vec4 0 0 0 20))
 
 (defvar *blocks* (list *ground* *block-a* *block-b* *block-c* *block-d*))
-
 ; Game logic
-(defmethod gamekit:draw ((app :moppu))
+(defmethod gamekit:draw ((app moppu))
+  (bodge-canvas:antialias-shapes nil)
```
Comment (borodust) [inline src/main.lisp:91]: This will remove AA tiling artifacts

File: src/main.lisp [hunk 4]
```diff
@@ -215,45 +221,51 @@
...
-  (lambda ()
-    (case *game-state*
-      (0
-        (setf *transitioning* t)
-      )
-
-      (1
-          (when *grounded*
+(defmethod post-initialize ((this moppu))
+  (gamekit:bind-button
```
Comment (borodust) [inline src/main.lisp:179]: All input binding is better to be placed inside `#'post-initialize`

File: .appveyor.yml [hunk 5]
```diff
@@ -44,7 +44,7 @@ deploy:
   tag: $(APPVEYOR_REPO_TAG_NAME)
   description: $(APPVEYOR_REPO_COMMIT_MESSAGE)
   auth_token:
-    secure: "tmQ/60JZBBCPZ8bahf5tp7meSUbUkHU68cssNPaoTN1+abcn+3a9G4pCwP957Dys"
+    secure: Z5XWjDOBlCrmfz3SQAjnLKtdgI5B2b/owJhRPNWYGrI+qwVNbBc4cZiroBZReWP7
```
Comment (srcnalt) [inline .appveyor.yml:47]: I created an access token in github and used that one, then should have used the exactly the same one in the documentation... 🤦‍♂

**Review by srcnalt — APPROVED** [review]

File: src/main.lisp (hunk 4 above)
Comment (srcnalt) [reply src/main.lisp:179]: This is what I have been loking for, trying to bind keys without start was not possible.

**Pull request merged**